src/stk/common/log.h

```diff
@@ -49,6 +49,7 @@ namespace stk
 {
     enum LogLevel
     {
+        Debug,
```

Comment:
I think I avoided using `Debug` as a log level because it kinda clashed with the `DLOG` macros (which are only evaluated for the debug build).

Maybe a `Verbose` level would be more suitable. I also think it is quite common for users of registration tools to want to see this mastodon wall of text with all information during the registration so it's not really debugging anyways.